collections/editor/occurrenceeditor.php

```diff
@@ -517,7 +517,6 @@ function requestImage(){
 	<script src="../../js/jquery.imagetool-1.7.js?ver=140310" type="text/javascript"></script>
 	<script src="../../js/symb/collections.editor.query.js?ver=6" type="text/javascript"></script>
 	<style type="text/css">
-		fieldset{ padding:15px }
```

Comment:
removing padding seems to create some overlapping in icons
<img width="643" alt="Screenshot 2024-07-01 at 11 42 49 AM" src="https://github.com/BioKIC/Symbiota/assets/86389284/fdff747f-006d-497b-a151-ec3f7c4c4f16">